### DISCUSSION TITLE
mcuboot 1.10.0  compiled binary size  16KB  is that expected?

### DISCUSSION THREAD

**Discussion by troycondit** [mcu-tools/mcuboot#1806] [General]
Im compiling mcuboot for baremetal Cortex M0+ processor (SAML21)  I need the UART support and with that enabled my compiled image size is 16032 bytes, I don't have a reference  of what the typical compiled size of MCU boot is so perhaps someone has some sizes they can share?

Thanks Troy

**Comment by Olstyle** [comment]
I need around 19kB using NDEBUG on an M33 with elliptic curves used for both signing and encryption. So it's in the same ballpark.
